### PR TITLE
Fixing unittest based on the changes of configuration

### PR DESCRIPTION
**Pull request by safwanrahman**
The unittest were failing for some reason. This PR will fix that
@glogiotatidis r?

### CODE REVIEW

**Comment by safwanrahman** [comment]
Now it pass `flake8`
r? @glogiotatidis

**Comment by glogiotatidis** [comment]
This is in my todo list, I'll review next monday.

File: kitsune/sumo/tests/test_googleanalytics.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def test_visitors_by_locale(self, _build_request):
         visits = googleanalytics.visitors_by_locale(
             date(2013, 01, 16), date(2013, 01, 16))
 
-        eq_(58, len(visits))
+        eq_(59, len(visits))
```
Comment (glogiotatidis) [inline kitsune/sumo/tests/test_googleanalytics.py]: Do you have more info why did this changed?

Comment (safwanrahman) [reply kitsune/sumo/tests/test_googleanalytics.py]: `Malay` locale was added in kitsune with #2888.
Therefore, there are total 59 locale.

Comment (glogiotatidis) [reply kitsune/sumo/tests/test_googleanalytics.py]: Hmmm OK but the test is limited to Jan 2013 which is before this change. Also this test doesn't seem to fail anymore

Comment (glogiotatidis) [reply kitsune/sumo/tests/test_googleanalytics.py]: I updated the test in #2965 with this change. Thanks!

**Comment by glogiotatidis** [comment]
Thank you for your time @safwanrahman. A lot of changes in this PR will be deprecated once #2965 gets merged. It also contains a lot of pep8 changes which are not required. 

Besides those it still has a number of useful improvements related to the search tests. I'd welcome a future PR -after #2965 gets merged- with those updates.

**Pull request closed without merging**